library/std/src/macros.rs

```diff
@@ -106,7 +106,7 @@ macro_rules! print {
 /// Use `println!` only for the primary output of your program. Use
 /// [`eprintln!`] instead to print error and progress messages.
 ///
-/// See [the formatting documentation in `std::fmt`](../std/fmt/index.html)
+/// See [the formatting documentation in `std::fmt`](../fmt/index.html)
```

Comment:
That would make it point to https://doc.rust-lang.org/nightly/fmt/index.html, which is not valid. The current code works fine in the rustdoc generated html. I think rust-analyzer will need to be changed to correctly handle this link instead.